### ISSUE TITLE
Is this really YOLOv4 ?

### ISSUE THREAD

**Issue by yuxx0218** [Tianxiaomo/pytorch-YOLOv4#249]
No CIOU Loss, no Lable Smoothing, no DropBlock Regularization, no Self-Adversarial Training, no DIOU NMS, no.......
It is defnitely a YOLO v3 replacing the backbone/neck/head.

**Comment by doduythao** [comment]
Maybe he's still working on it

**Comment by sumanthratna** [comment]
I think there is support for CIOU:
https://github.com/Tianxiaomo/pytorch-YOLOv4/blob/fe45fb64ba4c21130643f9b2569da7bb667356b3/train.py#L41-L127

**Comment by yuxx0218** [comment]
> 
> 
> I think there is support for CIOU:
> 
> https://github.com/Tianxiaomo/pytorch-YOLOv4/blob/fe45fb64ba4c21130643f9b2569da7bb667356b3/train.py#L41-L127

Thank you. I think you are right. Look forward to a complete version of YOLO v4.

**Comment by jpc** [comment]
It seems to me that `bboxes_iou` are used only for matching prediction and ground truth bboxes? The actual loss function seems to use only binary cross entropy and mse . Am I missing something here?
https://github.com/Tianxiaomo/pytorch-YOLOv4/blob/4ccef0ec8fe984e059378813e33b3740929e0c19/train.py#L265-L267
